random/module.py

```diff
@@ -0,0 +1,210 @@
+import aiohttp
+import random
+
+import discord
+from discord.ext import commands
+
+from core import utils, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Random(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def random(self, ctx, first: int, second: int = 0):
+        """Generate random number within the interval"""
+        if first > second:
+            first, second = second, first
+
+        await ctx.reply(random.randint(first, second))
+
+    @commands.cooldown(rate=3, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def pick(self, ctx, *args):
+        """Pick an option"""
+        for i, arg in enumerate(args):
+            if arg.endswith("?"):
+                args = args[i + 1:]
+                break
+
+        if not len(args):
+            return
+
+        option = utils.Text.sanitise(random.choice(args))
+        if option is not None:
+            await ctx.reply(option)
+
+    @commands.cooldown(rate=3, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def flip(self, ctx):
+        """Yes/No"""
+        choices = [_(ctx, "Yes"), _(ctx, "No")]
+        await ctx.reply(random.choice(choices))
+
+    @commands.cooldown(rate=5, per=20, type=commands.BucketType.channel)
+    @commands.command(aliases=["unsplash"])
+    async def picsum(self, ctx, *, seed: str = None):
+        """Get random image from picsum.photos"""
+        size = "900/600"
+        url = "https://picsum.photos/"
+        if seed:
+            url += "seed/" + seed + "/"
+        url += f"{size}.jpg?random={ctx.message.id}"
+
+        async with aiohttp.ClientSession() as session, session.get(url) as img_response:
+            if img_response.status != 200:
+                return await ctx.reply(f"E{img_response.status}")
+
+            image_id = str(img_response.url).split("/id/", 1)[1].split("/")[0]
+            async with session.get(f"https://picsum.photos/id/{image_id}/info") as response:
+                image_info = await response.json()
+
+        try:
+            image_url = image_info["url"]
+        except Exception:
+            image_url = discord.Embed.Empty
+
+        footer = "picsum.photos"
+        if seed:
+            footer += f" ({seed})"
+
+        embed = utils.Discord.create_embed(
+            author=ctx.author,
+            description=image_url,
+            footer=footer,
+        )
+        embed.set_image(url=str(img_response.url))
+
+        await ctx.reply(embed=embed)
+
+    @commands.cooldown(rate=5, per=20, type=commands.BucketType.channel)
+    @commands.command()
+    async def cat(self, ctx):
+        """Get random image of a cat"""
+        async with aiohttp.ClientSession() as session, \
```

Comment:
In this case please use second `using` statement inside, instead of using backslashes.
```py
async with ... as session:
    with ... as response:
        ...
```
Take using backslashes as a last resort, most of the problems can be solved differently. In this case, they break the visual indentation and the `if` block _inside_ the second `with` statement looks like it is _outside_ of it.